manim/animation/transform.py

```diff
@@ -359,21 +498,81 @@ def begin(self) -> None:
 
 
 class FadeToColor(ApplyMethod):
+    """Animation that changes color of a mobject.
+
+    Examples
+    --------
+
+    .. manim:: FadeToColorExample
+
+        class FadeToColorExample(Scene):
+            def construct(self):
+                self.play(FadeToColor(Text("Hello World!"), color=RED))
+
+    """
+
     def __init__(self, mobject: Mobject, color: str, **kwargs) -> None:
         super().__init__(mobject.set_color, color, **kwargs)
 
 
 class ScaleInPlace(ApplyMethod):
+    """Animation that scales a mobject by a certain factor.
+
+    Examples
+    --------
+
+    .. manim:: ScaleInPlaceExample
+
+        class ScaleInPlaceExample(Scene):
+            def construct(self):
+                self.play(ScaleInPlace(Text("Hello World!"), 2))
+
+    """
+
     def __init__(self, mobject: Mobject, scale_factor: float, **kwargs) -> None:
         super().__init__(mobject.scale, scale_factor, **kwargs)
 
 
 class ShrinkToCenter(ScaleInPlace):
+    """Animation that makes a mobject shrink to center.
+
+    Examples
+    --------
+
+    .. manim:: ShrinkToCenterExample
+
+        class ShrinkToCenterExample(Scene):
+            def construct(self):
+                self.play(ShrinkToCenter(Text("Hello World!")))
+
+    """
+
     def __init__(self, mobject: Mobject, **kwargs) -> None:
         super().__init__(mobject, 0, **kwargs)
 
 
 class Restore(ApplyMethod):
+    """Transforms a mobject to its last saved state.
```

Comment:
Do we really want an explicit example for Restore? There already is [this](https://docs.manim.community/en/stable/examples.html#followinggraphcamera) which demonstrates its use

Reply:
that example is mainly portraying how to make the camera follow. it may be difficult for new users to find it

Reply:
I agree, a separate example is fine.